exonum-java-binding/core/rust/integration_tests/tests/jni_cache_not_initialized.rs

```diff
@@ -14,10 +14,11 @@
 
 extern crate java_bindings;
 
-use java_bindings::utils::jni_cache;
+use java_bindings::utils::{assert_panics, jni_cache};
 
 #[test]
-#[should_panic(expected = "JNI cache is not initialized")]
 fn cache_not_initialized() {
-    jni_cache::runtime_adapter::execute_tx_id();
+    assert_panics("JNI cache is not initialized", || {
```

Comment:
There is no ambiguity in the test - I've just put it here in order to just make sure the function is accessible from the integration tests as well.